io.catenax.manufacturing_capability/3.0.0/gen/ManufacturingCapability-schema.json

```diff
@@ -1,363 +1,400 @@
 {
-  "$schema" : "http://json-schema.org/draft-04/schema",
```

Comment:
you need to remove this file also. only folder 3.1.0 is required.

Reply:
Folder 3.0.0 should not contain any changes anymore. Does this make the comment obsolete?